src/documentation/0052-nodejs-streams/index.md

```diff
@@ -112,13 +112,29 @@ There are four classes of streams:
 
 ## How to create a readable stream
 
-We get the Readable stream from the [`stream` module](https://nodejs.org/api/stream.html), and we initialize it
+We get the Readable stream from the [`stream` module](https://nodejs.org/api/stream.html), and we initialize it and implement the readable._read() method.
```

Comment:
Alternately, just escape the underscore, since the current diff makes it and emphasis
```suggestion
We get the Readable stream from the [`stream` module](https://nodejs.org/api/stream.html), and we initialize it and implement the `readable._read()` method.
```

Reply:
@nschonni fixed it. Thank you for your review.